packages/plugin/src/__tests__/ChartPlugin.spec.js

```diff
@@ -161,15 +154,6 @@ describe('ChartPlugin', () => {
             })
         })
 
-        describe('Year-on-year chart', () => {
-            beforeEach(() => {
-                props.visualization = {
-                    ...yearOverYearCurrentMock,
-                    option1: 'def',
-                }
-            })
-        })
```

Comment:
I've removed this as there is no test in the describe block and it was failing with the new jest version.